ci-operator/config/openshift-knative/operator/openshift-knative-operator-release-0.14.yaml

```diff
@@ -0,0 +1,31 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: "4.3"
```

Comment:
perhaps worth to go w/ 4.4 ?

Reply:
To be decided I think. As long as we don't know where we'll support 1.8 it's not worth changing imo.